pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -308,9 +308,9 @@ private boolean endCriteriaReached() {
             _consumeEndTime += TimeUnit.HOURS.toMillis(TIME_EXTENSION_ON_EMPTY_SEGMENT_HOURS);
             return false;
           }
-          _segmentLogger
-              .info("Stopping consumption due to time limit start={} now={} numRowsConsumed={} numRowsIndexed={}",
-                  _startTimeMs, now, _numRowsConsumed, _numRowsIndexed);
+          _segmentLogger.info(
```

Comment:
sorry for all the changes here. `Reformat Code` in the IDE did all this. happy to undo it if prefered

Reply:
Could you please undo the reformatting? It's a bit hard to follow the code change.

Reply:
done